vms/proposervm/block_test.go

```diff
@@ -92,12 +92,15 @@ func TestPostForkCommonComponents_buildChild(t *testing.T) {
 		vm:       vm,
 	}
 
+	emptyParentBlockSig := []byte{}
+
 	// Should call BuildBlockWithContext since proposervm is activated
 	gotChild, err := blk.buildChild(
 		context.Background(),
 		parentID,
 		parentTimestamp,
 		pChainHeight-1,
+		emptyParentBlockSig,
```

Comment:
style (optional): we do this pattern elsewhere in the codebase for values we hardcode into function params:

```suggestion
	// Should call BuildBlockWithContext since proposervm is activated
	gotChild, err := blk.buildChild(
		context.Background(),
		parentID,
		parentTimestamp,
		pChainHeight-1,
		[]byte{} /*=parentBlockVRFSig*/,
```

Reply:
If you'll scroll above, you'll see @marun 's comment on doing exactly the opposite:

> (No action required) Maybe use a constant (e.g. emptyParentBlockSig) instead of an empty value with a comment to declare your intent?

I'm quite neutral on which one to pick. Ideally, I'd like to be consistent.